modules/rosa-sdpolicy-networking.adoc

```diff
@@ -0,0 +1,65 @@
+
+== Cluster ingress
+Project administrators can add route annotations for many different purposes, including ingress control through IP whitelisting.
+
+Ingress policies can also be changed by using NetworkPolicy objects, which leverage the `ovs-networkpolicy` plugin. This allows for full control over the ingress network policy down to the pod level, including between pods on the same cluster and even in the same namespace.
+
+All cluster ingress traffic will go through the defined load balancers. Direct access to all nodes is blocked by cloud configuration.
+
+== Cluster egress
+Pod egress traffic control through `EgressNetworkPolicy` objects can be used to prevent or limit outbound traffic in {product-title}.
+
+Public outbound traffic from the master and infrastructure nodes is required and necessary to maintain cluster image security and cluster monitoring. This requires the 0.0.0.0/0 route to belong only to the internet gateway, it is not possible to route this range over private connections.
+
+OpenShift 4 clusters use NAT Gateways to present a public, static IP for any public outbound traffic leaving the cluster. Each availability zone a cluster is deployed into receives a distinct NAT Gateway, therefore up to 3 unique static IP addresses can exist for cluster egress traffic. Any traffic that remains inside the cluster, or does not go out to the public internet, will not pass through the NAT Gateway and will have a source IP address belonging to the node that the traffic originated from. Node IP addresses are dynamic, therefore a customer should not rely on whitelisting individual IP address when accessing private resources.
+
+Customers can determine their public static IP addresses by running a pod on the cluster and then querying an external service. For example:
+
+----
+oc run ip-lookup --image=busybox -i -t --restart=Never --rm -- /bin/sh -c "/bin/nslookup
+----
+
+== Cloud network configuration
+{Product-title} allows for the configuration of a private network connection through several cloud provider managed technologies:
+
+- VPN connections
+- AWS VPC peering
+- AWS Transit Gateway
+- AWS Direct Connect
+- Google Cloud VPC Network peering
+- Google Cloud Classic VPN
+- Google Cloud HA VPN
+
+[IMPORTANT]
+====
+Red Hat SRE's do not monitor private network connections. Monitoring these connections is the responsibility of the customer.
+====
+
+== DNS forwarding
+For {product-title} clusters that have a private cloud network configuration, a customer may specify internal DNS servers available on that private connection that should be queried for explicitly provided domains.
```

Comment:
s/may/can

Also, this sentence gets confusing in the last part...not sure how to understand "that should be queried", so perhaps this is clearer?

> ...a customer may specify internal DNS servers available on that private connection, which should be queried for explicitly provided domains.

Reply:
Since we're talking about an object and not a person, I believe it should be "..., that should be queried..." but I added the comma to make the sentence clearer. Also changed "may" to "can".